templates/static/contact-us.html

```diff
@@ -1,38 +1,70 @@
 {% extends 'layouts/_base.html' %}
 {% import 'macros/helpers.html' as helpers %}
+{% from "components/button/_macro.njk" import onsButton %}
 
-{% set survey_title  = survey_title or _("ONS Business Surveys") %}
+{% set survey_title = survey_title or _("ONS Business Surveys") %}
+{% set use_transactional_footer = true %}
 
 {% block main %}
   {% do page | setAttribute("feedback", "") %}
 
   <h1>{{ _("Contact us") }}</h1>
 
+  <p>{{ _("Contact us if you have any questions about filling in the census online or on paper.
+  You can use any of these options if you need information or a census in an alternative format like Braille or large print.") }}</p>
+  <br>
   <h2>{{ _("Telephone") }}</h2>
+  <p>{{ _("We will have advisers available to help you at the following times:") }}</p>
+  <ul class="list list--bare">
+    <li>{{ _("Monday to Friday - 8am – 7pm") }}</li>
+    <li>{{ _("Saturday - 8am - 1pm") }}</li>
+    <li>{{ _("Sunday - closed") }}</li>
+  </ul>
+
+  {% if schema_theme == "census-nisra" %}
+
+    <p>{{ _("Telephone: %(telephone_number)s",telephone_number = helpers.telephone_number(tel="0800 328 2021")) }}</p>
```

Comment:
We should stop using the `()%s` format as it could confuse the translators when our other strings are `{}`. All uses of this can be replaced with `_('').format(var=)`. So this would become:
```python
 _("Telephone: {telephone_number}").format(telephone_number = helpers.telephone_number(tel="0800 328 2021")) }}
```